users/serializers.py

```diff
@@ -83,23 +86,35 @@ def __init__(self, **kwargs: Any) -> None:
     )
 
     def validate_email(self, email: str) -> str:
+        logger.info("Custom register serializer - validate email method")
         email = allauth_get_adapter().clean_email(email)
         custom_validate_email_function(self, email)
         if allauth_settings.UNIQUE_EMAIL:
             if email and email_address_exists(email):
+                logger.warning(
+                    f"Email: {email} is already exist and throw exception with message: {CustomValidationErrorText.VALIDATION_ERROR_SIGNUP_EMAIL_MESSAGE} "
+                )
                 raise serializers.ValidationError(CustomValidationErrorText.VALIDATION_ERROR_SIGNUP_EMAIL_MESSAGE)
         return email
 
     @staticmethod
     def validate_password(password: str) -> str:
+        logger.debug(f"Validate password: {password}")
         return allauth_get_adapter().clean_password(password)
 
     def validate(self, data: Dict[str, Any]) -> Dict[str, Any]:  # pylint: disable=no-self-use
+        logger.debug(
+            f"Validate password and password confirmation with password: {data['password']} and password confirmation: {data['password_confirmation']}"
+        )
         if data["password"] != data["password_confirmation"]:
+            logger.warning(
```

Comment:
Not needed, it will happen all the time :)

Reply:
Done:)